SSA/Projects/InstCombine/Base.lean

```diff
@@ -119,12 +119,16 @@ def reprWithoutFlags (op : MOp.BinaryOp) (prec : Nat) : Format :=
     | .ashr   => "ashr"
     | .urem   => "urem"
     | .srem   => "srem"
-    | .add ⟨false, false⟩ => "add"
-    | .add ⟨nsw, nuw⟩    => toString f!"add {nsw} {nuw}"
-    | .mul    => "mul"
-    | .sub    => "sub"
-    | .sdiv   => "sdiv"
-    | .udiv   => "udiv"
+    | .add  ⟨false, false⟩ => "add"
+    | .add  ⟨nsw, nuw⟩     => toString f!"add {nsw} {nuw}"
+    | .mul  ⟨false, false⟩ => "mul"
+    | .mul  ⟨nsw, nuw⟩     => toString f!"mul {nsw} {nuw}"
+    | .sub  ⟨false, false⟩ => "sub"
+    | .sub  ⟨nsw, nuw⟩     => toString f!"sub {nsw} {nuw}"
```

Comment:
Why do we have a `toString` here? If we want to have a string, we can write it as `s!"..."`, but here the return type is `Format`, so just returning the result of `f!"..."` should be fine, I think